include/deal.II/numerics/vector_tools.h

```diff
@@ -919,13 +919,17 @@ namespace VectorTools
    * @endcode
    * where <code>qp_data</code> is a an object of type Table<2, VectorizedArray<double> >,
    * which stores quadrature point data.
+   *
+   * @p fe_component allow to additionally specify which component of @p data
+   * to use.
```

Comment:
You might want to slightly extend this description by referring more specifically to the `std::vector<const DoFHandler>` passed to the construction of the `MatrixFree` object and as set in the constructor of `FEEvaluation` - this way it gets more clear what the component refers to.

Reply:
@kronbichler i updated the docu, please have a look.

Reply:
Perfect.